rundeckapp/grails-app/views/framework/_editProjectForm.gsp

```diff
@@ -49,6 +49,30 @@
             <g:textField name="description" size="50"  value="${projectDescription}" class="form-control"/>
         </div>
     </div>
+    <div class="list-group-item">
```

Comment:
I think instead of hardcoding into the form, I will modify these config options to work via the `RundeckProjectConfigurable` interface.  (just making a note of this)